endpoints.json

```diff
@@ -25,5 +25,29 @@
         }
       ]
     }
+  },
+  "GET /api/articles/:article_id/comments": {
+    "description": "responds with an array of comments that match the article id",
+    "queries": ["article_id"],
+    "exampleResponse": {
```

Comment:
article_id is not a query here but a parametric endpoint which you have already showed with the : symbol, and this endpoint does not have any queries yet so this should be an empty array